============== FAILURE START =================
def test_02():
>       assert False
E       assert False


tests\test_me.py:6: AssertionError
============== FAILURE END =================
                                         cos
suitename2    testname2 filename2           
tests.test_me test_02   failing_01_.xml  1.0
                        failing_02_.xml  1.0
